src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/OrganisationIdentifierValidatorImpl.java

```diff
@@ -3,18 +3,28 @@
 import java.util.Collection;
 
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.dao.EmptyResultDataAccessException;
 import org.springframework.security.access.AccessDeniedException;
 import org.springframework.security.core.GrantedAuthority;
 import org.springframework.stereotype.Component;
 import org.springframework.util.StringUtils;
+import uk.gov.hmcts.reform.professionalapi.controller.advice.ResourceNotFoundException;
 import uk.gov.hmcts.reform.professionalapi.domain.Organisation;
 import uk.gov.hmcts.reform.professionalapi.domain.OrganisationStatus;
+import uk.gov.hmcts.reform.professionalapi.persistence.OrganisationRepository;
 import uk.gov.hmcts.reform.professionalapi.util.RefDataUtil;
 
 @Component
 @Slf4j
-public class OrganisationIdentifierIdentifierValidatorImpl implements OrganisationIdentifierValidator {
+public class OrganisationIdentifierValidatorImpl implements OrganisationIdentifierValidator {
```

Comment:
Validators should not have service/repository calls. All repository calls should be done via service.

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/444/commits/5266fb5901332c8363777707d56f6e1941266ac5